app/views/spree/admin/blogs/index.html.haml

```diff
@@ -0,0 +1,58 @@
+= content_for :page_title do
+  = Spree.t(:blogs)
+= content_for :page_actions do
+  = button_link_to Spree.t(:new_blog), new_object_url, class: "btn-success", icon: 'add'
+- if @blogs.any?
+  - content_for :table_filter_title do
+    = Spree.t(:search)
+  - content_for :table_filter do
+    %div(data-hook="admin_blogs_sidebar")
+      = search_form_for @search, url: 'blogs' do |f|
+        %div(data-hook="admin_blogs_index_search")
+          .clearfix
+            .col-md-6
+              .field
+                = f.label :name_cont, Spree.t(:name)
+                = f.text_field :name_cont, class: 'form-control'
+              .field.checkbox
+                %label
+                = f.check_box :enabled_true, include_hidden: false
+                = Spree.t(:enabled)
+            .col-md-12
+              .field.checkbox
+                %label
+                  = f.check_box :deleted_at_null, { checked: params[:q][:deleted_at_null] == '0' }, '0', '1'
+                  = Spree.t(:show_deleted)
+          %div.form-buttons.actions.filter-actions.text-center(data-hook="admin_blogs_index_search_buttons")
+            = button Spree.t(:search), 'icon-search'
+  %table.index.table.center#listing_bundles
+    %colgroup
+      %col(style='width: 10%;')
+      %col(style='width: 60%;')
+      %col(style='width: 10%;')
+      %col(style='width: 20%;')
+    %thead
+      %tr
+        %th(class='i')= sort_link @search,:id, Spree.t(:id), { default_order: 'desc' }, { title: 'admin_blogs_listing_id_title' }
+        %th= sort_link @search,:name, Spree.t(:name), {}, { title: 'admin_blogs_listing_name_title' }
+        %th= sort_link @search, :enabled, Spree.t(:enabled), {}, {title: 'admin_blogs_listing_enabled_title' }
+        %th.actions(data-hook='admin_blogs_index_header_actions')= Spree.t(:action_items)
+    %tbody
+      - @blogs.each do |blog|
+        %tr(id="#{spree_dom_id blog}" data-hook='admin_blogs_index_rows' class="#{cycle('odd', 'even')}")
+          %td= blog.id
+          %td= blog.name
+          %td
+            = form_for [:admin, blog], method: :put, remote: true, class: 'toggle_blog_enabled' do |form|
+              = form.check_box :enabled, class: 'blog_enabled'
+          %td.actions(data-hook='admin_blogs_index_row_actions')
+            = link_to_edit blog, no_text: true, class: 'edit' if can?(:edit, blog)
+            = link_to_delete blog, no_text: true, class: 'delete' if can?(:delete, blog)
+  = paginate @blogs
+- else
+  %p= Spree.t(:no_blogs)
+:javascript
```

Comment:
I think I'd added it in for enabling/disabling from the listing page itself, and didn't add the actual js piece in when I committed this. I think I've also fixed this with subsequent pushes.